backend/sentry.js

```diff
@@ -0,0 +1,25 @@
+const Sentry = require('@sentry/node')
+const sentryEventPrefix =
+  'https://sentry.io/organizations/origin-protocol/projects/dshop-backend/events'
+
+const { getLogger } = require('./utils/logger')
+const log = getLogger('sentry')
+
+const dsn = process.env.SENTRY_DSN
+const environment = process.env.ENVIRONMENT || 'development'
+
+if (dsn) {
+  log.info(`Initializing Sentry with environment: ${environment}`)
+  Sentry.init({ dsn, environment })
+} else {
+  if (environment === 'development') {
```

Comment:
Mentioned this in the other PR late, so not sure if you saw it, but why have this check at all?  Why not init sentry if `SENTRY_DSN` is defined?  Don't think we gain anything but an extra var to configure from this check.

Reply:
What I wanted to guard against is SENTRY_DSN not being set in production and us no noticing it...

Regarding the ENVIRONMENT env var, we do need it anyway to distinguish in Sentry between errors logged from staging vs prod.

Reply:
Read through this fast and only looked at the log statement that says "Skipping."  Message is a little misleading but the logic is all good.  :+1: